frontend/src/components/header/header.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 
+import { useGetWishlistsQuery } from '@store/queries/preferences/wishlist';
+
 import styles from './header.module.scss';
 import { Logo } from './logo/logo';
 import { Navigate } from './navigate/navigate';
 import { PrivateElements } from './private-elements/private-elements';
 import { PublicElements } from './public-elements/public-elements';
 
 export const Header: React.FC = () => {
+  const { data: wishlist } = useGetWishlistsQuery();
+
   const user = {
-    favorites: 3,
+    favorites: wishlist?.count as number | 0,
```

Comment:
`|` here is not what you expect :sweat_smile: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR